frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardPersonalData.yml

```diff
@@ -2,7 +2,7 @@ name: Safeguarding personal data
 question: >
   Do you agree to protect personal data according to the government guidance on <a href="https://www.gov.uk/data-protection-your-business" target="_blank" rel="noopener noreferrer">data protection and your business (opens in new tab)</a>?
 type: boolean
-hint: If you provide access to user research studios, for example, please confirm that you don’t share research videos with third parties.
+hint: If you provide access to user research studios, for example, please confirm that you do not share research videos with third parties.
```

Comment:
```suggestion
hint: If you provide access to user research studios, for example, can you confirm that you do not share research videos with third parties.
```

Reply:
Does this change to remove 'please' still make sense with the response options?

Reply:
Let's leave these changes to a separate PR